ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -103,6 +127,16 @@ module.exports = Marionette.LayoutView.extend({
     this.queryAdvanced.show(
       new FilterBuilderView({
         model: new FilterBuilderModel(),
+        suggester: async ({ id, type }) => {
```

Comment:
do you need `async` here?

Reply:
Since `fetchFacets` makes a network request, this needs to be async